tests/test-cases/Group6-VIC-Machine/6-17-Configure-TLS.robot

```diff
@@ -15,10 +15,9 @@
 *** Settings ***
 Documentation  Test 6-13 - Verify vic-machine configure TLS options
 Resource  ../../resources/Util.robot
-Suite Teardown  Run Keyword  Cleanup
+Suite Teardown  Run Keyword  Suite Cleanup
```

Comment:
Should we simply say this?

`Suite Teardown  Run Keyword  Cleanup VIC Appliance On Test Server`